capa/features/extractors/ida/helpers.py

```diff
@@ -167,6 +167,9 @@ def basic_block_size(bb):
 def read_bytes_at(ea, count):
     """ """
     segm_end = idc.get_segm_end(ea)
+    if segm_end == idaapi.BADADDR:
+        # address not in any segment
+        return 0
```

Comment:
is this better?

Reply:
Let's return `b""` here to maintain a consistent return type.

Reply:
ah, good point, let me address that... next week ;)